web/src/views/UserRegisterView/UserRegisterView.vue

```diff
@@ -1,11 +1,40 @@
 <template>
   <v-container>
-    TODO: UserRegisterView
+    <GirderRegister :oauthProviders="oauthProviders" />
   </v-container>
 </template>
 
 <script>
+import GirderRegister from '@girder/components/src/components/Authentication/Register.vue';
+import { OauthTokenPrefix, OauthTokenSuffix } from '@girder/components/src/rest';
+import girderRest, { loggedIn } from '@/rest';
+
 export default {
   name: 'UserRegisterView',
+  components: {
+    GirderRegister,
+  },
+  computed: {
+    loggedIn,
+  },
+  asyncComputed: {
+    async oauthProviders() {
+      return (
+        await girderRest.get('oauth/provider', {
+          params: {
+            redirect: `${window.location.href}${OauthTokenPrefix}{girderToken}${OauthTokenSuffix}`,
+            list: true,
+          },
+        })
+      ).data;
```

Comment:
I generally prefer destructuring for clarity, but I don't feel too strongly about it:
```suggestion
      const { data } = await girderRest.get('oauth/provider', {
        params: {
          redirect: `${window.location.href}${OauthTokenPrefix}{girderToken}${OauthTokenSuffix}`,
          list: true,
        },
      });

      return data;
```